tests/modules/individuals/resources/test_individual_crud.py

```diff
@@ -66,9 +66,7 @@ def test_create_read_delete_individual(db, flask_app_client):
         db.session.delete(temp_enc)
 
 
-@pytest.mark.skipif(
-    module_unavailable('individuals'), reason='Individuals module disabled'
-)
+@pytest.mark.skipif(extension_unavailable('edm'), reason='EDM extension disabled')
```

Comment:
why does this depend on edm?

Reply:
This is now semi defunct. This specific comment may be defunct but there are some tests that are disabled solely because there is no validation of the timeOf... vars and this needs adding